Error: Build error: Failed compiling object src/lcurlapi.o

:(
```
pi@raspberrypi:~/prog/KBot6-Lua $ sudo luarocks install Lua-cURL CURL_INCDIR=/usr/include/arm-linux-gnueabihf
Installing https://luarocks.org/lua-curl-0.3.10-1.src.rock
gcc -O2 -fPIC -I/usr/include/lua5.1 -c src/l52util.c -o src/l52util.o -I/usr/include/arm-linux-gnueabihf
gcc -O2 -fPIC -I/usr/include/lua5.1 -c src/lceasy.c -o src/lceasy.o -I/usr/include/arm-linux-gnueabihf
gcc -O2 -fPIC -I/usr/include/lua5.1 -c src/lcerror.c -o src/lcerror.o -I/usr/include/arm-linux-gnueabihf
gcc -O2 -fPIC -I/usr/include/lua5.1 -c src/lchttppost.c -o src/lchttppost.o -I/usr/include/arm-linux-gnueabihf
gcc -O2 -fPIC -I/usr/include/lua5.1 -c src/lcurl.c -o src/lcurl.o -I/usr/include/arm-linux-gnueabihf
gcc -O2 -fPIC -I/usr/include/lua5.1 -c src/lcutils.c -o src/lcutils.o -I/usr/include/arm-linux-gnueabihf
gcc -O2 -fPIC -I/usr/include/lua5.1 -c src/lcmulti.c -o src/lcmulti.o -I/usr/include/arm-linux-gnueabihf
gcc -O2 -fPIC -I/usr/include/lua5.1 -c src/lcshare.c -o src/lcshare.o -I/usr/include/arm-linux-gnueabihf
gcc -O2 -fPIC -I/usr/include/lua5.1 -c src/lcmime.c -o src/lcmime.o -I/usr/include/arm-linux-gnueabihf
gcc -O2 -fPIC -I/usr/include/lua5.1 -c src/lcurlapi.c -o src/lcurlapi.o -I/usr/include/arm-linux-gnueabihf
src/lcopturl.h: In function ‘lcurl_url_set_zoneid’:
src/lcurlapi.c:158:27: error: ‘CURLUPART_ZONEID’ undeclared (first use in this function); did you mean ‘CURLUPART_QUERY’?
   return lcurl_url_set(L, CURL##S);\
                           ^~~~
src/lcopturl.h:11:1: note: in expansion of macro ‘ENTRY_PART’
 ENTRY_PART(zoneid,        UPART_ZONEID   , CURLUE_UNKNOWN_PART    )
 ^~~~~~~~~~
src/lcurlapi.c:158:27: note: each undeclared identifier is reported only once for each function it appears in
   return lcurl_url_set(L, CURL##S);\
                           ^~~~
src/lcopturl.h:11:1: note: in expansion of macro ‘ENTRY_PART’
 ENTRY_PART(zoneid,        UPART_ZONEID   , CURLUE_UNKNOWN_PART    )
 ^~~~~~~~~~
src/lcopturl.h: In function ‘lcurl_url_get_zoneid’:
src/lcurlapi.c:168:27: error: ‘CURLUPART_ZONEID’ undeclared (first use in this function); did you mean ‘CURLUPART_QUERY’?
   return lcurl_url_get(L, CURL##S, E);\
                           ^~~~
src/lcopturl.h:11:1: note: in expansion of macro ‘ENTRY_PART’
 ENTRY_PART(zoneid,        UPART_ZONEID   , CURLUE_UNKNOWN_PART    )
 ^~~~~~~~~~
src/lcopturl.h: At top level:
src/lcurlapi.c:199:37: error: ‘CURLUPART_ZONEID’ undeclared here (not in a function); did you mean ‘CURLUPART_QUERY’?
   #define ENTRY_PART(N, S, E) { #S, CURL##S },
                                     ^~~~
src/lcopturl.h:11:1: note: in expansion of macro ‘ENTRY_PART’
 ENTRY_PART(zoneid,        UPART_ZONEID   , CURLUE_UNKNOWN_PART    )
 ^~~~~~~~~~

Error: Build error: Failed compiling object src/lcurlapi.o

```

--- a/src/lcopturl.h
+++ b/src/lcopturl.h
@@ -8,7 +8,10 @@ ENTRY_PART(query,         UPART_QUERY    , CURLUE_NO_QUERY        )
 ENTRY_PART(scheme,        UPART_SCHEME   , CURLUE_NO_SCHEME       )
 ENTRY_PART(url,           UPART_URL      , CURLUE_OK              )
 ENTRY_PART(user,          UPART_USER     , CURLUE_NO_USER         )
+
+#if LCURL_CURL_VER_GE(7,65,0)
 ENTRY_PART(zoneid,        UPART_ZONEID   , CURLUE_UNKNOWN_PART    )
+#endif
 
 ENTRY_FLAG(DEFAULT_PORT        )
 ENTRY_FLAG(NO_DEFAULT_PORT     )

